metron-platform/metron-writer/src/test/java/org/apache/metron/writer/hdfs/HdfsWriterTest.java

```diff
@@ -76,7 +76,8 @@ public void setup() throws IOException {
   public void testGetHdfsPathNull() {
     WriterConfiguration config = new IndexingWriterConfiguration(WRITER_NAME, new IndexingConfigurations());
     HdfsWriter writer = new HdfsWriter().withFileNameFormat(testFormat);
-    writer.init(new HashMap<String, String>(),createTopologyContext(), config);
+    writer.init(new HashMap<String, String>(), config);
+    writer.initFileNameFormat(createTopologyContext());
```

Comment:
Just for reference later - this is not part of the metron-common refactoring, but we should remove our Storm dependencies from the writer infrastructure as well.